org.brlcad.BRL_CAD.metainfo.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.brlcad.BRL_CAD</id>
+    <launchable type="desktop-id">org.brlcad.BRL_CAD.mged.desktop</launchable>
```

Comment:
this should match the app-id.

Reply:
BRL-CAD is a collection of smaller software, Mged, archer, rtwizard. With mged is where things usually start.
So I'm not sure what to do here. Because org.brlcad.BRL_CAD.desktop itself does not exist. Only org.brlcad.BRL_CAD.{mged,archer,rtwizard,db}.desktop exist.

Reply:
but you chose one `command`, didn't you?

Reply:
yeah, i guess i will go with mged then